File: 1895-minimum-number-of-operations-to-move-all-balls-to-each-box/1895-minimum-number-of-operations-to-move-all-balls-to-each-box.py
class Solution:
    def minOperations(self, boxes: str) -> List[int]:

        result=[0]*len(boxes)

        for i in range(len(boxes)):
            count=0
            for j in range(len(boxes)):
                if boxes[j]=='1':
                    count+=abs(j-i)
            result[i]=count
        return result
        

File: 1895-minimum-number-of-operations-to-move-all-balls-to-each-box/1895-minimum-number-of-operations-to-move-all-balls-to-each-box.ts
function minOperations(boxes: string): number[] {
    

    let result=[]

    for (let i=0 ; i<boxes.length;i++){
        let count=0
        for(let j=0;j<boxes.length;j++){
            if (boxes[j]=='1'){
                 count+=Math.abs(i-j)
            }
              
        } 
        result[i]=count
    }
    return result
};